helm/frost-server/templates/mqtt-deployment.yaml

```diff
@@ -62,17 +62,17 @@ spec:
             - name: mqtt_mqttServerImplementationClass
               value: "de.fraunhofer.iosb.ilt.sensorthingsserver.mqtt.moquette.MoquetteMqttServer"
             - name: mqtt_Enabled
-              value: "true"
```

Comment:
Setting `mqtt_Enabled` to `false` on the MQTT component makes no sense :)
Hence making this configurable is superfluous...

Reply:
I'm not shure what should be done to fix this. As far as I'm concerned there are two options:
* Just set this line to `value: "true"` again. This makes both the value `.Values.frost.mqtt.enabled` and the if-clause `{{ if .Values.frost.mqtt.enabled -}}` in `http-deployment.yaml` redundant.
* Set this line to `value: true` and delete the corresponding value and if-clause from http-deployment.

Reply:
Yes and no. The environment variable on the MQTT container is redundant, since if the MQTT container is started, MQTT is obviously supposed to be turned on. This enviroment variable is only used in the all-in-one container, where the MQTT part can be turned off.

The setting `.Values.frost.mqtt.enabled` should still be there, to control if the MQTT container should be started or not. This happens in the first line in `mqtt-deployment.yaml` and `mqtt-service.yaml`:
```{{- if .Values.frost.mqtt.enabled -}}```

In the http-deployment the if on line 62 is also required, since if MQTT is enabled, a message bus is required to allow communication between the HTTP and MQTT pods, while if MQTT is not enabled, this message bus is not needed.

Reply:
I haven't seen the `{{- if .Values.frost.mqtt.enabled -}}` at the start of `mqtt-deployment.yaml` - hence my confusion.
I have changed it back to "true".